desktop/apps/auction2/components/medium_filter/index.js

```diff
@@ -23,9 +26,16 @@ function MediumFilter(props) {
         checked={allMediumsSelected}
       />
       {
-        aggregatedMediums.map((agg) => {
-          const mediumSelected = contains(mediumIds, agg.id)
-          return <BasicCheckbox key={agg.id} item={agg} onClick={updateMediumParamsAction} checked={mediumSelected} />
+        _.map(mediumMap, (name, id) => {
+          const mediumSelected = contains(mediumIds, id)
+          const includedMedium = aggregatedMediums.find((agg) => { return agg.id === id })
```

Comment:
I would shorten to `.find(agg => agg.id === id)` unless you have a pref for this format?